_data/events.yml

```diff
@@ -33,10 +33,51 @@
   date: 2024-05-31
   track: "Bootcamp"
 
+  - name: "SpringBoot"
+  date: 2024-05-22
+  track: "ArchitectureTrack"
```

Comment:
Create a new track: `JavaTrack`
For the WebApi, this would be `DotNetTrack`

Also add this to the top of the file.